TWLight/resources/templates/resources/suggest.html

```diff
@@ -0,0 +1,107 @@
+   crossorigin="anonymous"></script>
+{% endblock head_javascript %}
+
+{% block content %}
+  {% if user.is_authenticated %}
+    <div class="panel panel-default full-width">
+      <div class="panel-heading">
+        {% comment %} Translators: This is the title of the form where users can submit partner suggestions. {% endcomment %}
+        <strong><span class="panel-title">{% trans 'Suggest partners' %}</span></strong>
+      </div>
+      <div class="panel-body">
+        {% crispy form %}
+      </div>
+    </div>
+  {% else %}
+    <div class="alert alert-warning">
+      {% url 'oauth_login' as the_url %}
+      {% comment %} Translators: This message is shown to users who visit the suggestions page and are not logged in. {% endcomment %}
+      {% blocktrans trimmed with request.path|urlencode as requested_path%}
+        To suggest partners, you should <strong><a href="{{ the_url }}?next={{ requested_path }}">log in</a></strong>.
+      {% endblocktrans %}
+    </div>
+  {% endif %}
+  {% if all_suggestions %}
+    <div class="panel panel-default full-width">
+      <div class="panel-heading">
+        {% comment %} Translators: This is the title of the panel where user suggestions are displayed. {% endcomment %}
+        <strong><span class="panel-title">{% trans 'Browse and support suggestions' %}</span></strong>
+      </div>
+      <div class="panel-body">
+        {% for suggest in all_suggestions %}
+          <div class="panel panel-default full-width">
+            <div class="panel-heading">
+              <div class="panel-title pull-left"><span class="panel-title">{{ suggest.suggested_company_name }}</span></div>
+              {% if user|coordinators_only %}
+                {% comment %} Translators: This button is at the partner suggestions page for coordinators to delete suggestions. {% endcomment %}
+                <a href="{% url 'suggest-delete' suggest.id %}" class="btn btn-outline-danger pull-right">{% trans 'Delete' %} <i class="fa fa-trash-o" aria-hidden="true"></i></a>
+              {% endif %}
+              <div class="clearfix"></div>
+            </div>
+            <div class="panel-body ">
+              <div class="col-md-10">
+                {% comment %} Translators: This is the text that appears before the description of every partner suggestion. {% endcomment %}
+                <p>
+                  <strong>{% trans 'Description' %}: </strong>
+                  {% comment %}
+                    Cache lifetime should be None, which would be forever since we're
+                    invalidating on update. However, as 2017-04-06 a bug in django
+                    doesn't allow for None to be passed from here into cache.  The
+                    fix is in master, but not release.
+                    https://code.djangoproject.com/ticket/27882
+                  {% endcomment %}
+                  {% cache 31536000 suggested_partner_description suggest.pk %} 

```

Comment:
This caching stuff is only need if you're using our twlight_wikicode2html mixin, eg ```suggestion.description | twlight_wikicode2html | safe```